### DISCUSSION TITLE
بهترین و دقیق ترین اسکنر با کانفیگ شخصی کدوم هستش؟

### DISCUSSION THREAD

**Discussion by Obeid1375** [MortezaBashsiz/CFScanner#436] [Q&A]
فکر کنم نسخه ویندوز زیاد دقیق نیست و پایتون از همه دقیق تر نتایجو میده درسته ؟
یا نسخه های دیگه ؟

**Comment by boyluti** [comment]
نسخه پایتون رو کجا باید استفاده کنیم ؟ چجوریه ؟ اموزشش
